ykcompile/src/lib.rs

```diff
@@ -273,6 +273,10 @@ impl<TT> CompiledTrace<TT> {
     fn exec_trace(&self, t_fn: fn(&mut TT) -> bool, args: &mut TT) -> bool {
         t_fn(args)
     }
+
+    pub fn ptr(&self) -> *const u8 {
```

Comment:
This might be dodgy unless `self.mc` explicitly guarantees a stable pointer at all points. Does it make such a guarantee?

Reply:
I'm not sure. The memory is allocated here: https://github.com/danburkert/memmap-rs/blob/master/src/unix.rs#L49
The rest is just a way to get `mc`, which is private, so that I can transmute the pointer in `PHASE_COMPILED`.

Reply:
OK, since this came from `mmap`, this should be fine. But let's add a comment explaining that, otherwise it'll scare future-me :)

Reply:
Fixed in 7dae376.